### ISSUE TITLE
Welcome and Quick Start Guide

### ISSUE THREAD

**Issue by msawired** [msawired/OpenProcessing-i18n#2]
For those I can not message privately [@deconbatch], here is my welcome message and Quick Start Guide: 

Thank you for joining! Your contributions will help Japanese-speaking users feel more welcome on OpenProcessing. Coding world is already full of unfriendly terms and lingo, and translations on OpenProcessing will highly increase accessibility and decrease the learning curve. 

Please check out the readme.md, and let me know what you think. This is the first time I am setting up an open source project on Github, so let me know if you have any thoughts on a better process. 

In short, you can create a branch, make changes on "lang.json", and test them on [beta.openprocessing.org](beta.openprocessing.org) via OP.loadLanguage('ja',[branchName]). When ready, create a pull request.
